manifests/router/link_route_pattern.pp

```diff
@@ -0,0 +1,20 @@
+# == Define: qpid::router::link_route_pattern
+#
+# Configure a link route pattern
+#
+# == Parameters
+#
+# $prefix::     Prefix to use
+#
+# $connector::  Connector for this link route pattern
+#
+define qpid::router::link_route_pattern(
+  $prefix    = 'queue.',
```

Comment:
I think you could have a prefix like `queue.pulp` and have it match `queue.pulp_foo` and `queue.pulp_bar`, so I'd prefer to just leave it as a free form field.